wee_slack.py

```diff
@@ -4101,6 +4130,56 @@ def command_thread(data, current_buffer, args):
 command_thread.completion = '%(threads)'
 
 
+def subscribe_helper(current_buffer, args, usage, api):
+    channel = EVENTROUTER.weechat_controller.buffers[current_buffer]
+    team = channel.team
+
+    if args:
+        msg = get_msg_from_id(channel, args)
+    elif isinstance(channel, SlackThreadChannel):
+        msg = channel.parent_message
+    else:
+        w.prnt('', usage)
+        return w.WEECHAT_RC_ERROR
+
+    s = SlackRequest(team, api,
+            {"channel": channel.identifier, "thread_ts": msg.ts}, channel=channel)
```

Comment:
You should print errors from this request. I.e. add `handle_subscriptionsthreadadd` and `handle_subscriptionsthreadremove` which does the equivalent of `handle_reactionsadd`.